bucket/determined.json

```
@@ -0,0 +1,16 @@
+{
+    "version": "0.26.7",
+    "description": "The Determined CLI is a command line tool that lets you launch new experiments and interact with a Determined cluster.",
+    "homepage": "https://github.com/sirredbeard/determined-windows-cli",
+    "license": "Apache-2.0",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/sirredbeard/determined-windows-cli/releases/download/0.27.0/determined-cli.zip",
+            "hash": "EF8B24D00AAE8C2019B7C80E1626CB69AAD03070E88E0F5055BA5CEDC16BE2EC"
+        }
+    },
+    "bin": "det.exe",
+    "checkver": {
+        "github": "https://github.com/sirredbeard/determined-windows-cli"
+    }
```

```suggestion
    "checkver": "github"
    }
```

No need to duplicate since github link already used as `homepage`.